### PR TITLE
Setup/upgrade deps 2

### PR DESCRIPTION
**Pull request by zirconias**
upgrading dependencies. see #5 

@vidbina  please can you accept merge and rebase other branches.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,29 +11,29 @@
...
   },
   "dependencies": {
-    "buffer": "^4.9.1",
+    "buffer": "^5.0.6",
     "intl": "^1.2.5",
     "moment": "^2.18.1",
-    "react": "^15.4.2",
-    "react-native": "^0.40.0",
-    "react-native-action-button": "^1.1.11",
+    "react": "15.4.1",
```
Comment (vidbina) [inline package.json:17]: We went form `^15.4.2` to `15.4.1` for `react` that's a regression. Why don't we bump to a higher version (is it due to a peer dependency issue?!?)

File: .gitignore [hunk 2]
```diff
@@ -48,6 +49,13 @@ ios/main.jsbundle
 # RVM artifacts
 .ruby-*
 
+# fastlane
```
Comment (vidbina) [inline .gitignore:52]: Were these comments auto generated?!?

**Review by vidbina — APPROVED** [review]
Just chatted about this. Due to a peer dependency issue react was bumped down.

File: .gitignore (hunk 2 above)
Comment (zirconias) [reply .gitignore:52]: generated by the rn upgrade command.  i choosed to put them to ignore this file in  future upgrades

**Pull request merged**